CONTRIBUTORS.md

```diff
@@ -1,4 +1,4 @@
-[![All Contributors](https://img.shields.io/badge/all_contributors-14-orange.svg?style=flat-square)](#contributors)
```

Comment:
![image](https://user-images.githubusercontent.com/26179770/50439464-279d5c00-0918-11e9-9544-12b5865a5e51.png)


Why have you changed number of contributors from 14 to 1?